docs/index.html

```diff
@@ -90,13 +93,13 @@ <h1 style="text-align: center;">GeoRust</h1>
                 projections.</td>
               <td class="px-4 py-3 text-indigo-900"><a href="https://github.com/georust/proj">GitHub</a></td>
               <td class="px-4 py-3 text-indigo-900"><a href="https://crates.io/crates/proj">crates.io</a></td>
-            </tr>
-            <tr>
-              <td class="px-4 py-3">Robust</td>
-              <td class="px-4 py-3 font-normal">Robust primitives for computational geometry.</td>
-              <td class="px-4 py-3 text-indigo-900"><a href="https://github.com/georust/robust">GitHub</a></td>
-              <td class="px-4 py-3 text-indigo-900"><a href="https://crates.io/crates/robust">crates.io</a></td>
-            </tr>
+              </tr>
```

Comment:
This seems wrong, but there a misplaced quote a couple of lines above, which probably confused your editor.

Reply:
Removed the `"`.